README.md

```diff
@@ -21,13 +21,13 @@ This filter passes a specified part of whole events to following filter/output p
       @type any_great_input
       @label @mydata
     </source>
-    
```

Comment:
I think this blank indentation is required to have a continuing code block.

Reply:
I was worried about that too, but it look correct in via github's  rich diff button:
![Screen Shot 2021-06-15 at 8 43 23 AM](https://user-images.githubusercontent.com/12193286/122083549-d1595780-cdb5-11eb-8f97-304b66a1584d.png)

Reply:
Wow, there should be a behavioral change.